tests/Integration/Endpoints/RestMetadataTest.php

```diff
@@ -43,14 +43,49 @@ public function set_up(): void {
 	}
 
 	/**
-	 * Test whether the logic has been enqueued in the `rest_api_init` hook with a filter that disables it.
+	 * Test whether the logic has been enqueued when the `run` method is called.
+	 *
+	 * @covers \Parsely\Endpoints\Rest_Metadata::run
+	 * @uses \Parsely\Endpoints\Rest_Metadata::register_meta
+	 */
+	public function test_register_enqueued_rest_init(): void {
+		global $wp_rest_additional_fields;
+
+		self::set_options( array( 'apikey' => 'testkey' ) );
+		self::$rest->run();
+
+		$this->assertParselyRestFieldIsConstructedCorrectly( 'page', $wp_rest_additional_fields );
+
+		// Cleaning up the registered fields.
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
+		$wp_rest_additional_fields = array();
+	}
+
+	/**
+	 * Test whether the logic has been enqueued when the `run` method is called with a filter that disables it.
 	 *
 	 * @covers \Parsely\Endpoints\Rest_Metadata::run
 	 */
 	public function test_register_enqueued_rest_init_filter(): void {
+		global $wp_rest_additional_fields;
+
```

Comment:
Maybe we should be setting an API key here just to prove the point that this works?